go/utils/restclient.go

```diff
@@ -17,8 +17,10 @@ import (
 // HTTPOK is ok status of http api call.
 const HTTPOK = "200 OK"
 
-var HttpClient = &http.Client{Transport: &http.Transport{}}
+// HTTPClient is a global Http client obj conains one persist connection.
+var HTTPClient = &http.Client{Transport: &http.Transport{}}
```

Comment:
Good point! I intended to make sure that the client can reuse the TCP connection for HTTP calls. I found this:

> A Client is an HTTP client. It's zero value (DefaultClient) is a usable client that uses DefaultTransport.

The Client's Transport typically has internal state (cached TCP connections), so Clients should be reused instead of created as needed. Clients are safe for concurrent use by multiple goroutines.

So `http.DefaultClient` seems to do the exact same thing!